### PR TITLE
docs: ref arch

### PR DESCRIPTION
**Pull request by Beilmann**
### Description

Replace this text with a summary of the changes in this PR. Include why the changes are needed and context about the changes. List required dependencies.

#### No release required

- [ ] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other changes that don't affect Terraform code

#### Release required

- [ ] Bug fix (patch release (`x.x.X`): Change that fixes an issue and is compatible with earlier versions)
- [ ] New feature (minor release (`x.X.x`): Change that adds functionality and is compatible with earlier versions)
- [ ] Breaking change (major release (`X.x.x`): Change that is likely incompatible with previous versions)

##### Release notes content

Replace this text with information that users need to know about the bug fixes, features, and breaking changes. This information helps the merger write the commit message that is published in the release notes for the module.

---

### Checklist for reviewers

- [ ] The PR references a GitHub issue.
- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Merge by using "Squash and merge".
- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author.

    The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).

### CODE REVIEW

**Review by surajsbharadwaj — DISMISSED** [review]

**Review by surajsbharadwaj — DISMISSED** [review]

**Comment by vburckhardt** [comment]
@SirSpidey - FYA, mostly documentation.

File: reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+subcollection: deployable-reference-architectures
+
+authors:
+  - name: Arnold Beilmann
+
+version: v7.0.0
+
+deployment-url: url #TODO
+
+docs: https://cloud.ibm.com/docs/solution-guide #TODO
```
Comment (SirSpidey) [inline reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md:17]: I think this docs link is correct, although the solution guides are not in the public docs yet.

File: reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+content-type: reference-architecture
+
+---
+
+{{site.data.keyword.attribute-definition-list}}
+
+# Power infrastructure for deployable architectures - variation 'PowerVS workspace'
+{: #deploy-arch-ibm-pvs-inf-standard}
+{: toc-content-type="Power infrastructure for deployable architectures - variation 'PowerVS workspace'"}
```
Comment (SirSpidey) [inline reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md:43]: content type attribute should be `{: toc-content-type="reference-architecture"}`

File: reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+{: toc-industry="Technology"}
+{: toc-use-case="ITServiceManagement"}
+{: toc-compliance="SAPCertified"}
+
+Building upon the VPC service that were previously created when you deployed Secure infrastructure on VPC for regulated industries, Power Infrastructure for deployable architectures creates a Power Virtual Server workspace and connects it with VPC services. Proxy service for public internet access from PowerVS workspace is configured.
+
+Additional management components (NFS server, NTP forwarder, and DNS forwarder) may be installed on VPC optionally.
```
Comment (SirSpidey) [inline reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md:50]:
```suggestion
You can optionally install more management components (such as NFS server, NTP forwarder, and DNS forwarder) on VPC.
```

Making it active voice and replacing a couple of words that we try not to use (additionally, may).

File: reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+![Architecture diagram for 'Power infrastructure for deployable architectures' - variation 'PowerVS workspace'.](deploy-arch-ibm-pvs-inf-standard.svg "Architecture diagram"){: caption="Figure 1. Single-zone PowerVS workspace accessable over secure landing zone" caption-side="bottom"}
+
+## Design requirements
+{: #design-requirements}
+
+![Design requirements for 'Power infrastructure for deployable architectures' - variation 'PowerVS workspace'.](heatmap.svg "Design requirements"){: caption="Figure 1. Scope of the solution requirements" caption-side="bottom"}
+
+IBM Cloud® Power Virtual Servers (PowerVS) is a public cloud offering that lets an enterprise establish its own private IBM Power computing environment on shared public cloud infrastructure. PowerVS is logically isolated from all other public cloud tenants and infrastrucutre components, creating a private, secure place on the public cloud. The reference architecture for 'Power infrastructure for deployable architectures' - variation 'PowerVS workspace' is designed to provide a framework for building a PowerVS offering according to the best practices and requirements using IBM Cloud® deployable architectures framework.
```
Comment (SirSpidey) [inline reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md:62]:
```suggestion
IBM Cloud® Power Virtual Servers (PowerVS) is a public cloud offering that an enterprise can use to establish its own private IBM Power computing environment on shared public cloud infrastructure. PowerVS is logically isolated from all other public cloud tenants and infrastructure components, creating a private, secure place on the public cloud. This reference architecture provides a framework to build a PowerVS offering according to the best practices and requirements from the IBM Cloud® deployable architectures framework.
```

fixes typo: infrastrucutre, avoids the anthropomorphism (product lets you) and simplifies the last sentence by shortening it.

File: reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+|* create virtual server instance to host basic management services like DNS, NTP, NFS|Management services VPC instance|CreateLinux VPC instance that may host management components. Preconfigure ACL and security group rules to allow communication for basic management componets (NFS - ports XXX, NTP - ports XXX, DNS - ports XXX)|Modify number of virtual server instances and allowed ports in preset or perform the modifications manually|
+|* ensure financial services compliancy for VPC services  \n * perform network setup of all created services  \n * perform network isolation of all created services  \n * ensure all created services are inteconnected with each other|Secure landing zone components|Create a minimum set of required components for a secure landing zone|Create a modified set of required components for a secure landing zone in preset|
```
Comment (SirSpidey) [inline reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md:79]:
```suggestion
|* Ensure public internet connectivity  \n * Isolate most virtual instances to not be reachable directly from the public internet|Edge VPC service|Create a separate VPC service where public internet connectivity is allowed to be configured| |
|* Provide infrastructure administration access  \n * Limit the number of infrastructure administration entry points to ensure security audit|Management VPC service|Create a separate VPC service where SSH connectivity from outside is allowed| |
|* Provide infrastructure for service management components like backup, monitoring, IT service management, shared storage  \n * Ensure you can reach all IBM Cloud and on-premises services|Workload VPC service|Create a separate VPC service as an isolated environment, without direct public internet connectivity and without direct SSH access| |
|* Allow customer to choose operating system from two most widely used commercial Linux operating system offerings  \n * Support new OS releases|Linux operating system|Red Hat Enterprise Linux (RHEL)|SUSE Linux Enterprise Server(SLES)|
|* Create a virtual server instance that can act as an internet proxy server|Proxy server VPC instance|Create a Linux VPC instance that can act as a proxy server. Preconfigure ACL and security group rules to allow public internet traffic over proxy using default proxy ports (3828)|Configure application load balancer to act as proxy server manually|
|* Create a virtual server instance as the only management access point to the landscape|Bastion host VPC instance|Create a Linux VPC instance that acts as a bastion host. Configure ACL and security group rules to allow SSH connectivity (port 22). Add a public IP address to the VPC instance. Allow connectivity from a restricted and limited number of public IP addresses. Allow connectivity from IP addresses of the Schematics engine nodes||
|* Create a virtual server instance to host basic management services like DNS, NTP, NFS|Management services VPC instance|Create a Linux VPC instance that can host management components. Configure ACL and security group rules to allow communication for basic management components (NFS - ports XXX, NTP - ports XXX, DNS - ports XXX)|Modify number of virtual server instances and allowed ports in preset or perform the modifications manually|
|* Ensure financial services compliancy for VPC services  \n * Perform network setup of all created services  \n * Perform network isolation of all created services  \n * Ensure all created services are interconnected |Secure landing zone components|Create a minimum set of required components for a secure landing zone|Create a modified set of required components for a secure landing zone in preset|
```

- Fix some typos
- Initial caps
- remove `pre` from configure

File: reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+|-------------|-----------|--------------------|--------------------|
+|* connect PowerVS workspace with VPC services|Cloud connections|Setup two redundant cloud connections|                    |
+|* configure network for management of all the instances  \n * troughput and latency are not relevant|Management network|Configure private network with default configurations and attch it to both cloud connections|                    |
+|* configure separate network for backup purposes with higher data throughput|Backup network|Configure separate private network with default configurations and attch it to both cloud connections. Networks characteristics might be adapted by the users manually (e.g., to improve throughput)|                    |
+|* preload OS images relevant for customer workload|Pre-loaded OS images|Preload Linux OS images for SAP workload. Keep the number of preloaded images at minimum to save corsts.|Modify the input parameter that specify the list of preloaded OS images.|
+|* preload public SSH key that will be injected into every OS deployment|Pre-loaded SSH public key|Preload customer specified SSH public key|                    |
```
Comment (SirSpidey) [inline reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md:91]:
```suggestion
|* Connect PowerVS workspace with VPC services|Cloud connections|Set up two redundant cloud connections| |
|* Configure the network for management of all instances  \n * Throughput and latency are not relevant|Management network|Configure private network with default configurations and attach it to both cloud connections| |
|* Configure separate network for backup purposes with higher data throughput|Backup network|Configure separate private network with default configurations and attach it to both cloud connections. Networks characteristics might be adapted by the users manually (for example to improve throughput)| |
|* Pre-load OS images relevant for customer workload|Pre-loaded OS images|Pre-load Linux OS images for SAP workload. Keep the number of pre-loaded images at minimum to save costs.|Modify the input parameter that specifies the list of pre-loaded OS images.|
|* Pre-load a public SSH key that will be injected into every OS deployment|Pre-loaded SSH public key|Pre-load customer specified SSH public key| |
```

- Fix typos
- Make `pre-load` consistent (prefer `preload` or `install` but assume some of this is coming from other sources.
- replace Latin expression `e.g.`

File: reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+{: caption="Table 2. PowerVS workspace architecture decisions" caption-side="bottom"}
+
+### PowerVS management services architecture decisions
+{: #vpc-components}
+
+| Requirement | Component | Choice | Alternative choice |
+|-------------|-----------|--------------------|--------------------|
+|* ensure public internet connectivity from all the instances to be deployed in PowerVS workspace|SQUID proxy|Setup SQUID proxy software on Linux virtual server instance runnning in edge VPC|                    |
```
Comment (SirSpidey) [inline reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md:99]: Can you make the same changes in the following tables?
- initial capitalization
- replace e.g., with For example
- fix typos (`runnning`, `Sinchronize` and `sinchronisation`)

File: reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+|* provide a DNS forwarder to a DNS server not directly reachable from PowerVS workspace (e.g., running on-premise or in other isolated environment)|DNS forwarder|Configure DNS forwarder on Linux virtual server instance runnning in workload VPC| By using native IBM Cloud DNS service, DNS forwarder is not needed. Direct domain name resolution is possible.|
+{: caption="Table 3. PowerVS management services architecture decisions" caption-side="bottom"}
+
+### Network security architecture decisions
+{: #vpc-components}
+
+| Requirement | Component | Choice | Alternative choice |
+|-------------|-----------|--------------------|--------------------|
+|* isolate edge VPC and allow only a limited number of network connections  \n * all other connections from/to edge VPC are forbidden|ACL and security group rules in edge VPC|Open following ports by default:|Additional ports might be opened in preset or added manually after deployment|
```
Comment (SirSpidey) [inline reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md:110]: - initial caps
- missing port? `Open following ports by default:`

File: reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+|* preload VPN configuration to simplify VPN setup|VPNs|VPN configuration is the responsibility of the customer|                    |
+{: caption="Table 4. Network security architecture decisions" caption-side="bottom"}
+
+### Key and password management architecture decisions
+{: #vpc-components}
+
+| Requirement | Component | Choice | Alternative choice |
+|-------------|-----------|--------------------|--------------------|
+|* use public/private SSH key to access virtual server instances via SSH  \n * use SSH proxy to login on all virtual server instances via the bastion host  \n * do not store private ssh key on any virtual instances, also not on the bastion host  \n * do not keep any SSH login posibilities other than specified private/public SSH key pair|public SSH key - provided by customer. private SSH key - provided by customer.|As customer to specify the keys. Accept the input as secure parameter or as reference in IBM Cloud Secure Storage Manager. Do not print SSH keys in any log files. Do not persist private SSH key.|                    |
```
Comment (SirSpidey) [inline reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md:122]: - initial caps
- typo: `posibilities`

File: reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+
+## Compliance
+{: #compliance}
+
+This reference architecture is certified for SAP deployments.
+
+## Next steps
+{: #next-steps}
+
+As next step, you install deployable architecture 'SAP on Power' on top of that infrastructure.
```
Comment (SirSpidey) [inline reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md:133]:
```suggestion
Install the SAP on Power deployable architecture on this infrastructure.
```

**Review by SirSpidey — CHANGES_REQUESTED** [review]
Some suggestions, mostly in the tables. 

Also want to add a link to the reference architecture in the main readme file, as explained in the module template: https://github.com/terraform-ibm-modules/terraform-ibm-module-template/blame/main/README.md#L86-L92

**Pull request closed without merging**